converged_sound_open_fw_modules_development/FW/src/intel_adsp/build_framework/build_core/ace_extlib_meu_conf.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<CodePartition version="2.5" >
+    <Name value="ADSP" help_text="Name to use in the output binary's directory. Maximum length is 4 characters. " />
+    <Length value="0x0" help_text="Length of output binary, extra space will be filled with 0xFF's. If length is smaller than required, an error will be reported. If set to 0, the length will be computed as needed by the tool." />
+    <Usage value="audioImage0Manifest" value_list="CseBupManifest,,CseMainManifest,,PmcManifest,,UsbTypeCIOMManifest,,UsbTypeCNphyManifest,,UsbTypeCTBTManifest,,WcodManifest,,LoclManifest,,IntelUtokManifest,,SPHYManifest,,PchcManifest,,SamfManifest,,PphyManifest,,GbstManifest,,BootPolicyManifest,,iUnitBootLoaderManifest,,iUnitMainFwManifest,,audioImage0Manifest,,audioImage1Manifest,,IfwiManifest,,OsBootLoaderManifest,,OsKernelManifest,,OemSmipManifest,,IshManifest,,OemDebugManifest,,OemKeyManifest,,SilentLakeVmmManifest,,OemDnxIfwiManifest" help_text="Indicates the type of data contained in this binary. This value is used during signature verification to validate the public key." />
+    <VendorId value="0x8086" />
```

Comment:
The value_list should probably be omitted here. We just need to specify the value. @mwasko please check